src/components/common/keyUtils.tsx

```diff
@@ -19,3 +19,15 @@ export function cancelEvent(e) {
   e.preventDefault();
   e.stopPropagation();
 }
+
+export function getScrollParent(node) {
+  if (node == null) {
```

Comment:
In cases where the node is undefined, I suggest using the following syntax to avoid using a non-strict comparison
```
if(!node){
   return null;
}
```